charts/traction/templates/ui/configmap.yaml

```diff
@@ -29,6 +30,7 @@ data:
   SERVER_SMTP_SECURE: {{ .Values.ui.smtp.secure | quote }}
   SERVER_SMTP_USER: {{ .Values.ui.smtp.user | quote }}
   SERVER_TRACTION_URL: http://{{ include "tenant_proxy.fullname" . }}:{{ .Values.tenant_proxy.service.port }}
+  SERVER_LOKI_URL: http://{{ include "loki.fullname" . }}:{{ .Values.loki.service.port }}
```

Comment:
`loki.fullname` will not be available unless Loki is deployed at the same time as Traction.
I think it might be best to just use a full URL from the values file in case Loki is not being deployed with the solution. Otherwise you'll have to add Loki as a dependent chart to Traction and handle on/off scenarios.